rooms.py

```diff
@@ -47,28 +64,34 @@ def room_allocations(self, args):
             for staff in staff_occupancy:
                 living_space_allocations[room_name].append(staff[-1])
 
-        divider = max(max([len(", ".join(i)) for i \
-            in office_space_allocations.values() if i[0] is not None]),
-            max([len(", ".join(i)) for i \
-            in living_space_allocations.values() if i[0] is not None]))
+        try:
+            office_space = [
+                len(", ".join(i)) for i in office_space_allocations.values() if i[0] is not None]
```

Comment:
**Code Quality - Nitpick:** Fix the line length here and in the other areas identified during the review.